lib/kubernetes-deploy/kubernetes_resource/daemon_set.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 require 'kubernetes-deploy/kubernetes_resource/pod_set_base'
+
 module KubernetesDeploy
   class DaemonSet < PodSetBase
     TIMEOUT = 5.minutes
+    PREFETCH_KINDS_ON_SYNC = %w(Pod Node)
```

Comment:
I actually think Node should not be prewarmed. It is only needed on the very first sync, so letting that one get delayed by an inline fetch seems ok to me.